tox.ini

```diff
@@ -3,7 +3,7 @@ envlist =
     py{35,36,37}-django20
     py{35,36,37}-django21
     py{35,36,37}-django22
-    py{35,36,37}-django30
+    py{36,37,38}-django30
```

Comment:
This change is not related to Mixer, can you please file a separate PR if changes are needed here?

Reply:
I've revered this change in this PR.  At least locally, this was causing tox to fail (since python 3.5 isn't in the supported python versions for Django 3.0), but I have no idea if this will affect the travis build or not.